managed/ui/src/components/universes/UniverseConnectModal/UniverseConnectModal.js

```diff
@@ -124,6 +124,13 @@ class UniverseConnectModal extends Component {
       const {
         universeDetails: { clusters, communicationPorts }
       } = universeInfo;
+
+      const tlsAvailability = getFeatureState(
+        currentCustomer.data.features,
+        'universes.details.overview.manageEncryption'
+      );
+      const isTLSEnabled = isEnabled(tlsAvailability);
```

Comment:
The way to check if TLS is enabled for a universe is the metadata of the universe details.
Redux state:
From `state.universe.currentUniverse.data.universeDetails.clusters`, get the primary cluster's `userIntent` and check whether `enableNodeToNodeEncrypt` or `enableClientToNodeEncrypt` flags are set to true. If either flag (or both) are set to true, TLS is enabled for the universe.

Reply:
@yb-andrew : I have already fixed it the right way in https://github.com/yugabyte/yugabyte-db/pull/9473, so when this PR: 9441 is merged first and then when [9473](https://github.com/yugabyte/yugabyte-db/pull/9473/files) is merged, we should not have any confusion. Please let me know. Thanks.